sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -223,6 +223,44 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
     fromSequenceNumber?: Long;
 }
 
+// @public
+export interface ProcessErrorContext {
+    entityPath: string;
+    errorSource: "complete"
+    /**
+     * Covers errors that occur when the user (or autoComplete) abandons a message.
+     */
+     | "abandon"
+    /**
+     * Errors thrown from the user's `processMessage` callback passed to `subscribe`
+     */
+     | "userCallback"
+    /**
+     * Errors thrown when receiving messages.
+     */
+     | "receive"
+    /**
+     * Errors thrown when the user calls renewLock or when the internal lock renewer calls renewLock.
+     * Automatic lock renewal can be enabled via the `CreateReceiverOptions.maxAutoLockRenewalDurationInMs`
+     * property passed when calling `ServiceBusClient.createReceiver()`
+     */
+     | "renewLock"
+    /**
+     * Broadly covers a series of phases - the actual initialization that occurs when an operation
+     * is first started as well as errors that occur when reconnecting.
+     */
+     | "initialize"
+    /**
+     * Errors thrown when accepting a session.
+     */
+     | "acceptSession"
+    /**
+     * Errors thrown when closing a session.
+     */
+     | "closeSession";
```

Comment:
In .Net, the `acceptSession` and `closeSession` are more applicable because they support working with multiple sessions where they may pick and drop sessions often. We don't have this feature, do these error sources still apply?

Reply:
I do use 'acceptSession' in one spot but closeSession doesn't really get used. 

The one spot it does happen would probably fall under 'initialize' if we keep that flag or 'receive' if we don't. So getting rid of this seems easy enough.

Reply:
The error source is only used by the `subscribe()` method right? When working with sessions, the 
`acceptSession()` method takes care of the initialization and subscribe() doesnt ever have to. Am I missing something?

Reply:
There was one spot where we have an "else" that says "hey, if this session is closed it must be because we didn't open it". That was the only spot I used it.

I never used closeSession  and it's all gone now and subsumed into 'receive' as a category now.